client/src/i18n/fr/posting_journal.json

```diff
@@ -26,31 +26,34 @@
       "COLUMNS" : "Colonnes"
     },
     "ERRORS": {
-      "DATE_IN_WRONG_PERIOD" : "Dates dans la mauvaise periode",
-      "MISSING_DESCRIPTION" : "Descriptions omises",
-      "NO_FISCAL_FOR_DATE" : "Il n'existe pas d'année fiscale pour cette Date",
       "CLOSED_FISCAL_YEAR" : "L'annee fiscale pour cette date a déjà été fermé",
-      "VARIOUS_TRANSACTION_TYPE" : "Type de transaction different pour une même transaction",
-      "TRANSACTION_DIFF_DATES" : "Une transaction doit avoir des dates identiques, mais certaines transactions contiennent des dates différentes. Corrigez cela en mettant à jour les dates des transactions affectées.",
+      "CREDITED_DEBITED": "Sur une même ligne de transaction un compte ne peut pas être débité et crédité en même temps",
+      "DATE_IN_WRONG_PERIOD" : "Dates dans la mauvaise periode",
+      "DEACTIVATED_USER" : "Une transaction a été effectuée par un compte d'utilisateur qui a été désactivé. Veuillez lire attentivement cette transaction, déverrouiller le compte d'utilisateur et publier.",
+      "DEB_CRED_NOT_NULL" : "Le débit et le crédit ne doivent pas être égale à Zero",
+      "EDIT_INVALID_ACCOUNT" : "Transaction avec des comptes non valides",
+      "EDIT_INVALID_ENTITY" : "Transaction avec une entité non valide",
+      "EDIT_INVALID_REFERENCE" : "Transaction avec référence non valide",
       "LOCKED_ACCOUNT" : "Compte(s) verrouille(s)",
+      "LOCKED_CREDITOR_GROUP_ACCOUNT" : "Ces transactions utilisent un compte d'un groupe de créditeurs verrouillé et peuvent avoir été faites par erreur. Pour corriger cela, modifiez les transactions ou déverrouillez le groupe de créditeurs.",
+      "LOCKED_DEBTOR_GROUP_ACCOUNT" : "Ces transactions utilisent un compte d'un groupe de débiteurs verrouillé et peuvent avoir été faites par erreur. Pour corriger cela, modifiez les transactions ou déverrouillez le groupe de débiteurs.",
       "LOCKED_PERIOD" : "Periodes(s) verrouille(s)",
-      "UNBALANCED_TRANSACTIONS" : "Transaction(s) non balancée(s)",
       "MISSING_ACCOUNTS" : "Compte(s) omis",
-      "CREDITED_DEBITED": "Sur une même ligne de transaction un compte ne peut pas être débité et crédité en même temps",
-      "DEB_CRED_NOT_NULL" : "Le débit et le crédit ne doivent pas être égale à Zero",
       "MISSING_DATES" : "Date(s) manquante(s)",
+      "MISSING_DESCRIPTION" : "Descriptions omises",
       "MISSING_DOCUMENT_ID" : "Ligne(s) sans ID Document",
       "MISSING_ENTITY" : "Ligne(s) sans ID entite",
       "MISSING_FISCAL_OR_PERIOD" : "Année fiscale ou période omise",
+      "MISSING_TRANSACTIONS" : "Aucune transaction n'a été soumise.",
       "MULTIPLE_CANCELLING" : "Annulation multiple. Cette transaction a deja été annulée",
+      "NO_FISCAL_FOR_DATE" : "Il n'existe pas d'année fiscale pour cette Date",
       "SINGLE_LINE_TRANSACTION" : "Transaction(s) avec une ligne",
```

Comment:
Try to use "Une transaction doit comporter deux lignes ou plus. Pour corriger cela, localisez la transaction et ajoutez la ligne d'équilibrage." for key "SINGLE_LINE_TRANSACTION"

Reply:
Merci!